docs/src/main/sphinx/connector/jdbc-procedures.fragment

```diff
@@ -12,13 +12,28 @@
 
 - `system.execute('query')`
 
-  The `execute` procedure allows you to query the underlying database directly.
-  This can be useful for accessing native features which are not available in Trino
-  or for executing DDL or DML that `query` pass-through table function doesn't support.
-  For example, the following system call drops the default value from `your_column`
-  on `your_table` table in the `example` catalog
+  The `execute` procedure allows you to execute a query in the underlying data
+  source directly. The query must use supported syntax of the connected data
+  source. Use the procedure to access features which are not available in Trino
+  or to execute queries that return no result set and therefore can not be used
+  with the `query` or `raw_query` pass-through table function. Typical use cases
+  are statements that create or alter objects, and require native feature such
+  as constraints, default values, automatic identifier creation, or indexes.
+  Queries can also invoke statements that insert, update, or delete data, and do
+  not return any data as a result.
+
+  The query text is not parsed by Trino, only passed through, and therefore only
+  subject to any security or access control of the underlying data source.
+
+  For example, the following system call drops the default value from
+  `your_column` on `your_table` table in the `example` catalog in a connected
+  PostgreSQL database using the PostgreSQL SQL syntax in the parameter value
```

Comment:
This fragment goes to each connector. Readers might be confused why PostgreSQL is mentioned in different connectors. I feel that there's no need to mention the connector name because DROP DEFAULT is SQL standard syntax.

Reply:
Well, we somehow need to clarify in the example that the SQL is whatever the data source supports .. not Trino SQL .. so if we just use the standard .. then its not as clear.

I know its kinda confusing to have a PostgreSQL SQL example in the Snowflake connector or so .. but having a different example in all of them might be a bit much. Although we kinda do that for the query table function.

I will try to reword.

Reply:
I think I got a feasible solution.. please check it out.